info/views.py

```diff
@@ -16,7 +19,7 @@ def index(request):
         return render(request, 'info/t_homepage.html')
     if request.user.is_student:
         return render(request, 'info/homepage.html')
-    return render(request, 'info/logout.html')
+    return render(request, 'info/admin_page.html')
```

Comment:
We should only return this page if the user is a superuser and return the logout page otherwise because there could a user that is neither student, teacher or superuser. 

```suggestion
    if request.user.is_superuser:
        return render(request, 'info/admin_page.html')
    return render(request, 'info/logout.html')
```